src/Tools/dotnet-counters/Exporters/ConsoleWriter.cs

```diff
@@ -126,9 +130,15 @@ public void AssignRowsAndInitializeDisplay()
         {
             Clear();
             
+            _consoleWidth = Console.WindowWidth;
+            _consoleHeight = Console.WindowHeight;
+            _maxNameLength = _consoleWidth < 80 ? _consoleWidth - CounterValueLength: 60; // truncate the display name when the window width is less than 80
```

Comment:
What happens when _consoleWidth < 20? I don't think dotnet-counters needs to produce useful output in such a tiny window, but ideally we shouldn't crash and things should look good once the user makes the window bigger again. It looks like a small _consoleWidth would make _maxNameLength negative, which would trigger an Exception when MakeFixedWidth() at 155 calls string.Substring() with negative length?

Reply:
I made the following change to make sure _maxNameLength is never negative:
```c#
_maxNameLength = Math.Max(Math.Min(80, _consoleWidth) - (CounterValueLength + Indent + 1), 0);
```

When _consoleWidth < 20, it will now only print out the numerical value. The output doesn't look great because at that small of console width the values begin to wrap lines, but the output does look better when the console is made bigger again.

Reply:
what does the +1 do in the calculation? If console width = 80, Indent = 4, CounterValueLength = 20 I was anticipating _maxNameLength = 56.

Reply:
The +1 is for the space between the name and the values, which can be seen here.:

```c#
Console.WriteLine($"{name} {FormatValue(counter.LastValue)}");
```

 Also, I initially included the indent and additional space between the name and the value into CounterValueLength, so having CounterValueLength = 20 was incorrect. I changed to the correct value of CounterValueLength = 15, meaning that Indent = 4, CounterValueLength = 15, + 1 for the extra space. Therefore, at console width = 80, _maxNameLength = 60.